language/tools/move-package/tests/test_sources/nested_deps_git_local/Move.resolved

```diff
@@ -55,6 +55,22 @@ ResolvedGraph {
             "MoveStdlib",
             "NestedDeps",
         },
+        fetched_deps: {
+            Git(
+                GitInfo {
+                    git_url: "https://github.com/move-language/move",
+                    git_rev: "781c844",
+                    subdir: "language/move-stdlib",
```

Comment:
The `subdir` should not be part of your cache key, only the URL of the remote, and its revision.  `download_and_update_if_remote` already calculates (in effect) a kind of string-based cache key, to form the directory name to download the remote dependencies into, I would suggest re-using that.

Reply:
I re-used the entire `DependencyKind` as it seemed like we may have to distinguish between `Custom` and `Local` variants in case they happen to point to the same location, and simply using enum as the key seemed simplest. Looking more closely, particularly at the `repository_path` function whose result could be used at key, it seems like we don't have to worry about that.